src/wiring-hapi.js

```diff
@@ -19,6 +19,7 @@ server.connection({
 
 const swaggerOptions = {
   jsonEditor: true,
+  reuseDefinitions: false,
```

Comment:
if some models are the same it reuses them in the final generated json to reduce the generated file on big APIs.
As I had issues with model labels not generated correctly for cases with structures like `schema: swagger.List(swagger.Team)` (only on the first swagger.List generated the labels, on the others it didn't, although it was the same definition) I disabled reuse. This didn't solve the problem but I went with disabling optimizations until it generates proper output.